t/cobrand/tfl.t

```diff
@@ -264,6 +267,7 @@ subtest "creating a user on TfL creates tfl_password extra_metadata" => sub {
         my $tfl_user = FixMyStreet::DB->resultset("User")->find({ email => $email});
         ok $tfl_user->get_extra_metadata('tfl_password'), "TfL encrypted password created";
         $mech->log_out_ok();
+        undef $tfl_mock;
```

Comment:
Doesn't hurt, but just to note I think the mock should delete itself when it goes out of scope.